pallets/did/src/benchmarking.rs

```diff
@@ -96,6 +96,9 @@ where
 	<T as Config>::Currency: Mutate<T::AccountId>,
 {
 	let balance = <CurrencyOf<T> as Inspect<AccountIdOf<T>>>::minimum_balance()
+		+ <T as Config>::BaseDeposit::get()
```

Comment:
But why do we bump it up so much?

Reply:
A bunch of new DIDs are created since the last release. Therefore, the fees are higher. While benchmarking, I got the error that the account has not enough balance to pay for them. I increased them by an arbitrary number.

Reply:
Wait, what do you mean? Isn't the fee fixed based on the space a DID takes? How is the number of DIDs created affecting this?

Reply:
No, I am not talking about the did deposit. I was thinking the Tx fees are higher because of the new created dids.